tv_shows/index.html

```diff
@@ -0,0 +1,39 @@
+<html>
+    <head>
+        <title>TV Shows App</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+        <div id="page-container">
+            <h1>TV Shows App</h1>
+            <div id="show-preview">
+                <img id="dark-image" src="images/dark_background.png" alt="Dark" width="100%" />
+                <div id="show-description">
+                    <h2>Dark</h2>
+                    <div>
+                        A family saga with a supernatural twist, set in a German town where the disappearance of two young children exposes the relationships among four families.
+                    </div>
+                    <div id="average-rating"></div>
+                </div>
+            </div>
+            <div id="reviews">
```

Comment:
Za ovo Stjepka krivi, od njega smo ovo pokupili 😄 
Između multiline blokova stavimo malo prostora, da bude čitljivije
```suggestion
            </div>

            <div id="reviews">
```